api/src/main/java/com/exelaration/abstractmemery/controllers/MemeController.java

```diff
@@ -21,4 +24,11 @@ public Meme uploadMeme(@RequestBody Meme meme) throws Exception {
 
     return memeService.save(meme);
   }
+
+  @GetMapping("/memes")
```

Comment:
The URL specification here is unnecessary.  Used properly, REST specifies that a `GET` call with no parameters is a "get all" request.  Thus, the URL for this request should be `/meme/`, but this line makes it `/meme/memes/`.